webapp/channels/src/components/search/search.tsx

```diff
@@ -207,6 +207,14 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         handleUpdateSearchTerms(pretextArray.join(' '));
     };
 
+    const handleUpdateSearchTeam = async (teamId: string) => {
+        await actions.updateSearchTeam(teamId);
+        handleSearch().then(() => {
+            setKeepInputFocused(false);
+            setFocused(false);
+        });
```

Comment:
Yeah the await was not required there. I decided to leave the `then()` here because it's done this way above, and I think it makes it clear that those actions are the results of the search being done